### ISSUE TITLE
failing when cache-point is 0.4.1

### ISSUE THREAD

**Issue by danikaze** [jsdoc2md/dmd#49]
I've been trying to debug this error for several hours
```
node_modules/jsdoc2md-stats/lib/jsdoc2md-stats.js:96
        throw err
        ^
Error: The partial docs could not be found
```

after I compared my packages with another working project, I saw a difference on one of the dependencies version (`cache-point`).
It looks like it works fine with 0.4.0, but not 0.4.1
I had to add this to my `package.json`:
```
"resolutions": {
  "cache-point": "0.4.0"
}
```

so I think it's better to change `package.json` to use `cache-point@0.4.0` for now (instead of `cache-point@^0.4.0`)

**Comment by 75lb** [comment]
hi, thanks for the report.. Does it fail in all cases, with all code? Will review this ASAP

**Comment by danikaze** [comment]
well, the error was not related to the code to document, but to the setting of the environment?
I'm not sure why it couldn't find the partials (even the .hbs files are there), but it fixed after using 0.4.0 of this package.

Thanks for your prompt answer

**Comment by 75lb** [comment]
i can't reproduce it, could you post your OS, node, npm and terminal version please. I'm on node v8.4.0, npm v5.4.1, macOS 10.12.6, iTerm2.

The only change between cache-point 0.4.0 and 0.4.1 was to upgrade the underlying version of `mkdirp`.. I don't believe this is what's causing the `The partial docs could not be found` error. I think it's a red herring and if i can't reproduce i can't fix it. 

Could you post a reproduction case? A code snippet or gist..
